Example/Controllers/SwiftUI/Features/Settings/Features/MiniAppSettingsContactsView.swift

```diff
@@ -15,7 +15,8 @@ struct MiniAppSettingsContactsView: View {
                     ContactCellView(
                         name: contact.name ?? "",
                         contactId: contact.id,
-                        email: contact.email ?? ""
+                        email: contact.email ?? "",
+                        allEmails: contact.allEmailList ?? [""]
```

Comment:
```suggestion
                        allEmails: contact.allEmailList ?? [contact.email ?? ""]
```

Reply:
I think doing this will include primary email in the list of additional emails. Which will have the redundant data in the list. And may create the confusion when using the additional emails list. This will also change the behavior of the allEmailList as per my understanding. and this list will always have one value.

Reply:
I suggest you go with an empty list then `[]`

Reply:
Yes this worked nicely now I am using [] instead of [""]. Thank you. :)